app/src/main/java/com/jvoye/tasky/agenda/domain/MockAgendaRepository.kt

```diff
@@ -0,0 +1,19 @@
+package com.jvoye.tasky.agenda.domain
+
+import com.jvoye.tasky.core.domain.TaskyItemId
+import com.jvoye.tasky.core.domain.model.TaskyItem
+import com.jvoye.tasky.core.domain.util.DataError
+import com.jvoye.tasky.core.domain.util.EmptyResult
+import kotlinx.coroutines.flow.Flow
+
+interface MockAgendaRepository {
```

Comment:
This looks more like a mock interface than something we’d want in the production code 😅. Maybe it’s worth moving into the test source set **(or a fake/ package)** so we keep a clean separation between real contracts and testing. If you notice you’re reaching for this pattern often, it could also be a good nudge to add some proper tests around these flows instead of keeping mocks wired into prod. What do you think?